### PR TITLE
[Edge] new function to invoke event callback

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Add new function to invoke event and export edge-event functions.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: gst/nnstreamer/tensor_query/nnstreamer_edge_common.c [hunk 1]
```diff
@@ -124,6 +124,67 @@ nns_edge_event_get_type (nns_edge_event_h event_h, nns_edge_event_e * event)
...
+    return NNS_EDGE_ERROR_INVALID_PARAMETER;
+  }
+
+  if (!data_h) {
+    nns_edge_loge ("Invalid param, data_h should not be null.");
+    return NNS_EDGE_ERROR_INVALID_PARAMETER;
+  }
+
+  if (ee->event != NNS_EDGE_EVENT_NEW_DATA_RECEIVED) {
+    nns_edge_loge ("Invalid param, data_h should not be null.");
```
Comment (gichan-jang) [inline gst/nnstreamer/tensor_query/nnstreamer_edge_common.c:150]: Please update the error message.

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_query/nnstreamer_edge_common.c]: updated

File: gst/nnstreamer/tensor_query/nnstreamer_edge_internal.c [hunk 2]
```diff
@@ -99,6 +99,52 @@ static int _nns_edge_create_message_thread (nns_edge_handle_s * eh,
...
+  ret = nns_edge_event_create (event, &event_h);
+  if (ret != NNS_EDGE_ERROR_NONE) {
+    nns_edge_loge ("Failed to create new edge event.");
+    return ret;
+  }
+
+  if (data) {
+    ret = nns_edge_event_set_data (event_h, data, data_len, destroy_cb);
+    if (ret != NNS_EDGE_ERROR_NONE) {
+      nns_edge_loge ("Invalid event data, failed to create new edge event.");
```
Comment (gichan-jang) [inline gst/nnstreamer/tensor_query/nnstreamer_edge_internal.c:133]: Please update the error message.

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_query/nnstreamer_edge_internal.c]: updated

**Review by gichan-jang — APPROVED** [review]
LGTM!

**Review by anyj0527 — APPROVED** [review]
LGTM✔

**Pull request merged**